test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -170,7 +176,9 @@ private static void createAndShowGUI() {
         btnStart.requestFocus();
 
         mainFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        mainFrame.setLayout(new BoxLayout(mainFrame.getContentPane(), BoxLayout.Y_AXIS));
+        mainFrame.getContentPane()
+                 .setLayout(new BoxLayout(mainFrame.getContentPane(),
+                                          BoxLayout.Y_AXIS));
```

Comment:
Minor nit:
Can be simplified by removing .getContentPane() from ln#170-173 as well.

```suggestion
        mainFrame.setLayout(new BoxLayout(mainFrame.getContentPane(),
                                          BoxLayout.Y_AXIS));
```

Reply:
This not necessary… it doesn't change the logic, and leaving adding components as they are (unless updated with a new vertical box) preserves attribution to the original author of the test.

Minor nit, the suggestion is incorrect, or rather it looks incorrect. The first parameter to `BoxLayout` constructor should be the component to which you set the layout, and preserving `.getContentPane()` makes the code look correct.